compiler/il/Aliases.cpp

```diff
@@ -797,6 +797,10 @@ OMR::SymbolReference::setSharedShadowAliases(TR_BitVector * aliases, TR::SymbolR
       while (symRef = i.getNext())
          if (symRef->getSymbol() == self()->getSymbol())
             aliases->set(symRef->getReferenceNumber());
+
+      // include symbol reference's own shared alias bitvector
```

Comment:
Okay that's good in the sense that aliasing between JitBuilder generated sym refs for unions will work (meaning that languages whose IL gen uses JitBuilder will work fine). 

One last question : Is it worth considering the setting of really shares symbol flag inside of makeSharedAliases so that other languages that call the general makeSharedAliases API on two sym refs actually get the answer they expect when they get the aliases ?